django_seo_js/middleware/hashbang.py

```diff
@@ -17,7 +17,7 @@ def process_request(self, request):
         if "_escaped_fragment_" not in request.GET:
             return
 
-        url = request.build_absolute_uri()
+        url = self.backend.build_absolute_uri()
```

Comment:
This all completely breaks because I forgot to pass request in. Will update in a bit.